Packages/vcs/Lib/vtk_ui/manager.py

```diff
@@ -41,6 +49,14 @@ def __render(self, obj, event):
             self.timer = None
             self.window.Render()
 
+    def widget_at_point(self, x, y):
+        picker = vtk.vtkPropPicker()
```

Comment:
I decided to let the Manager determine if there are any widgets at a point, rather than just manually checking out its renderers in the Configurator; leaves more wiggle room for later.